### PR TITLE
Add `shutdown-interval` CLI argument

### LINKED ISSUE #54: Hupper fails to reload app cleanly on Docker Compose environments
Hello all, first off thanks for hupper 🌟 

The warehouse team uses hupper in our development stack but [recently](https://github.com/pypa/warehouse/issues/6578) we started noticing the it failed to restart on code changes. I believe this is because we invoke hupper in a Docker Compose environment causes the `input` call to fail with an EOF.

We've worked around it by pinning the hupper version to 1.7 but ideally there'd be a CLI option to prevent reading from stdin. Would that be acceptable?

### PR DESCRIPTION
**Pull request by yeraydiazdiaz**
Fixes https://github.com/Pylons/hupper/issues/54

### CODE REVIEW

**Comment by digitalresistor** [comment]
Please sign https://github.com/Pylons/hupper/blob/master/CONTRIBUTORS.txt

@mmerickel this needs a fix in `tox.ini` to always download the latest version of pip because it can't install pytest at the moment.

**Comment by mmerickel** [comment]
I can't see a reason to support a float here versus an integer. Sorry for the slowness in reviewing but would you mind making it more strict for now?

File: src/hupper/cli.py [hunk 1]
```diff
@@ -6,13 +6,28 @@
...
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-m", dest="module", required=True)
     parser.add_argument("-w", dest="watch", action="append")
     parser.add_argument("-x", dest="ignore", action="append")
     parser.add_argument("-v", dest="verbose", action='store_true')
     parser.add_argument("-q", dest="quiet", action='store_true')
+    parser.add_argument(
+        "-i", "--shutdown-interval", type=shutdown_interval_parser
```
Comment (mmerickel) [inline src/hupper/cli.py]: I don't think `-i` should be reserved for this feature just yet. A future `--reload-interval` has as much right to that one as this and it isn't clear and obvious enough to use. Would you mind dropping the `-i`?

**Comment by mmerickel** [comment]
Thank you!

**Pull request merged**